.gitignore

```diff
@@ -0,0 +1,55 @@
+# Compiled documentation
+propagation/SWALS/doc
+
+# Test outputs
+propagation/SWALS/tests/*/OUTPUTS
+propagation/SWALS/tests/*/*.log
+propagation/SWALS/tests/*/*unit_tests
+propagation/SWALS/tests/unit_tests/*.csv
+propagation/SWALS/tests/unit_tests/*.nc
+
+# Example outputs
+propagation/SWALS/examples/**/OUTPUTS
```

Comment:
Here and below, how is the `**` notation different from `*`

Reply:
An asterisk "*" matches anything except a slash.
A slash followed by two consecutive asterisks then a slash matches zero or more directories.

Reply:
Ahh, so this version would also match  `propagation/SWALS/examples/OUTPUTS`, whereas a single `*` would not?

Reply:
Yes I believe so, and it would also match `propagation/SWALS/examples/my_folder/OUTPUTS` as well as  `propagation/SWALS/examples/my_folder/another_folder/OUTPUTS`